honeybee_energy/load/ventilation.py

```diff
@@ -74,6 +74,7 @@ def __init__(self, identifier, flow_per_person=0, flow_per_area=0, flow_per_zone
         self.flow_per_zone = flow_per_zone
         self.air_changes_per_hour = air_changes_per_hour
         self.schedule = schedule
+        self._user_data = None
```

Comment:
Remove this line since it already happens when initializing the parent class.

Reply:
commit: loads cleaned up